### DISCUSSION TITLE
Ivy telescope theme for find notes and search notes?

### DISCUSSION THREAD

**Discussion by megalithic** [nvim-telekasten/telekasten.nvim#222] [Q&A]
Might there be a way to set the telescope theme for the find notes and search notes commands? i'd like to set it to `ivy`.

**Comment by lambtho12** [comment]
There is no direct built-in option in Telekasten for that.

However, it might still be possible. You can try crafting some mapping for the find_notes command that passes the require theme option. See [Telescope readme about themes](https://github.com/nvim-telescope/telescope.nvim#themes)

**Reply by lambtho12** [reply]
I have not tested, but maybe something along these lines:
```lua 
--In your init.lua

vim.keybind.set("n", "<leader>zf", ":lua require('telekasten').find_notes(require('telescope.themes').get_ivy({}))<CR>")
```
